pkg/sdk/metrics/metrics.go

```diff
@@ -0,0 +1,74 @@
+package metrics
+
+import (
+	"sync"
+
+	prom "github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	eventsMetricName     = "operator_event_types_total"
+	syncEventsMetricName = "operator_reconcile_results_total"
```

Comment:
Can you also rename the variables for the `reconcile_results` metric accordingly.
`syncEventsMetricName ` ==> `reconcileResultsMetricName`
Also below
`SyncResultSuccess` ==> `ReconcileResultSuccess`
`SyncResultFailure` ==> `ReconcileResultFailure`